docs/contributing/coding-style.rst

```diff
@@ -1,43 +1,55 @@
 Coding Style
 ============
 
+Saleor uses the `pre-commit <https://pre-commit.com/#install>`_ tool to check and
+automatically fix any formatting issue before creating a git commit.
+
+You can easily install it after having setup saleor by running the below instruction.
```

Comment:
It's a good practice to avoid words like "easily" in docs like this, as they're highly subjective (though I know it's easy ;)). I would be more specific here: "Run the following command to install pre-commit into your git hooks and have it run on every commit:"

It would be nice to also say below something like: "See ".pre-commit-config.yaml" for pre-commit configuration.